### ISSUE TITLE
DFL 2022 | Resources - Article link grid

### ISSUE THREAD

**Issue by kevinhowbrook** [mozilla/foundation.mozilla.org#9402]
### Describe the feature request
We'll need a block type to house the multiple Article link blocks created in #9401.

### Screenshots / mock-ups
![Screenshot(10)](https://user-images.githubusercontent.com/2128707/191294611-502c6bd9-18b3-4f73-bb83-766f13dee2e7.png)

### Additional context
Link to designs

---

**Dev Tasks:** 
* [ ] Create new article link grid block allowing editors to add up to 3 article links
* [ ] Configure article link grid block template

**Acceptance Criteria:**
- [ ] Design to look as per Figma files
- [ ] As an editor creating a banner page I can add an article link grid block consisting of 3 links to article pages
- [ ] Template stacks verticle on mobile

**Comment by cdanfon** [comment]
Same as per comment on #9401 - Can we move this to 'ready to deploy' and label it as 'blocked'? Also, could you post here the PRs that are blocking it for reference please?

**Comment by cdanfon** [comment]
Same as per comment on #9401 - Can we move this to 'ready to deploy' and label it as 'blocked'? Also, could you post here the PRs that are blocking it for reference please?

**Comment by kevinhowbrook** [comment]
This was accidentally merged. New PR is here https://github.com/mozilla/foundation.mozilla.org/pull/9562

It's blocked by https://github.com/mozilla/foundation.mozilla.org/pull/9511

**Comment by tbrlpld** [comment]
@cdanfon Ready for test on staging

**Comment by cdanfon** [comment]
Awaiting on #9401 fixes, however upon first inspection looks like the section is also missing the line above the card



![Image](https://user-images.githubusercontent.com/102978346/198351086-6818e02f-a27d-44e2-baa1-ca0224a23012.png)
![Image](https://user-images.githubusercontent.com/102978346/198351087-84566de8-54bc-40c5-b662-5aa3d49bc520.png)

**Comment by stevedya** [comment]
I've added a PR for these fixes: 
I've added the border fix to this PR https://github.com/mozilla/foundation.mozilla.org/pull/9565 

- Adds missing borders
- Adjusts link hover underline to match styleguide

This is ready for you to look at now @tbrlpld

**Comment by tbrlpld** [comment]
@cdanfon This was merged and deployed to staging yesterday.

**Comment by gutermul** [comment]
Design looks good. There is is just still the outstanding functionality to being an article page picker rather than a news feature block where we can select news articles from the wider internet. See comment in #9401

**Comment by cdanfon** [comment]
Thanks @gutermul! As per #9401 are you happy to mark this as complete based on the original scope of the ticket please?

**Comment by cdanfon** [comment]
After speaking with @gutermul we've agreed to mark this as completed, however Lisa would need to have a chat with the rest of the DFL team to understand if ticket https://github.com/mozilla/foundation.mozilla.org/issues/9585 needs to be addressed

**Comment by cdanfon** [comment]
This is now on prod. Closing as completed
